services/blockchain-indexer/shared/database/schema/votes.js

```diff
@@ -14,10 +14,9 @@
  *
  */
 module.exports = {
-	primaryKey: 'tempId',
+	primaryKey: ['id', 'receivedAddress'],
 	schema: {
-		// TODO: Remove 'tempId' after composite PK support is added
-		tempId: { type: 'string' }, // ['id', 'receivedAddress']
+		tempId: { type: 'string' },
```

Comment:
Please remove the `tempId` column. We do not need it any more with the composite primary key support.
```suggestion
```